source/docs/guides/3D_secure.md

```diff
@@ -177,7 +198,7 @@ system variables:
 | Attribute | Description |
 | --- | --- |
 | xid | Include the 3D Secure transaction identifier (up to 20-digits). |
-| eci | SecureECI is a 1-digit status code: 5 – authenticated; 6 – attempted, not completed. |
+| eci | SecureECI is a 1-digit status code: 5 â€“ authenticated; 6 â€“ attempted, not completed. |
```

Comment:
Dashes not showing in browser properly

Reply:
Oh shoot those were supposed to be dashes!